workers/data_refinery_workers/downloaders/sra.py

```diff
@@ -297,11 +298,23 @@ def download_sra(job_id: int) -> None:
 
     original_file = original_files.first()
     sample = original_file.samples.first()
-    if _has_unmated_reads(sample.accession_code):
-        original_files = _replace_dotsra_with_fastq_files(sample, job, original_file)
 
     downloaded_files = []
     success = None
+
+    # Try accessing FTP server, see if that was successful
+    try:
+        if _has_unmated_reads(sample.accession_code, job):
+            original_files = _replace_dotsra_with_fastq_files(sample, job, original_file)
+    except:
```

Comment:
Can you make sure it's an issue with connecting to the server:
```suggestion
    except ftplib.all_errors:
```
(from https://stackoverflow.com/a/3169751/6095378)

Reply:
(you'll also need to import ftplib instead of FTP.